### PR TITLE
Create AltStore repository

### PR DESCRIPTION
**Pull request by bazzadazza72**
I would like to propose a contribution in the form of a custom [AltStore](https://altstore.io/) repository which will make installation easier until the iOS app can be published to the App Store (whenever that ends up happening).

The repository currently does not have any apps listed in it, however, I'd be good to add a release once we get iOS compilation working properly.

I can also create some documentation to teach users how to add the repository to their devices, let me know if you want that done too.

Screenshot of the repository in its current state:

IMG_0132

Edit: Here's the source documentation if you're interested: https://faq.altstore.io/developers/make-a-source

### CODE REVIEW

**Comment by PartyDonut** [comment]
That's a good idea would make it easier for iPhone users to side-load the app.
Could you also mark this PR as a draft so we can keep better track of what should or should not be reviewed entirely? 
Or atleast I'm assuming the json needs more information to properly work?

File: altstore.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "PartyDonut's Repository",
+  "subtitle": "A source for Fladder and co.",
+  "iconURL": "https://avatars.githubusercontent.com/u/42371342",
+  "tintColor": "#F54F32",
```
Comment (PartyDonut) [inline altstore.json]: The current icon background is #3a2101

**Review by PartyDonut — CHANGES_REQUESTED** [review]

**Comment by bazzadazza72** [comment]
My apologies for going radio silent over the past couple days.

After a large amount of troubleshooting (and many commits as you can see), I have finally gotten the repository to work (I use AltServer in a VM that isn't powered on right now, so I'll test if the app installs later).

If you want to try before merging, here's the link to the repository: https://raw.githubusercontent.com/bazzadazza72/fladder-develop/refs/heads/develop/altstore.json

**Review by PartyDonut — APPROVED** [review]
Looks good to me thanks for working on this.

Do you know if the altStore app supports a button similar to playstore/appstore? Would be great to display on the landing page on github.

**Comment by krinkuto11** [comment]
> Looks good to me thanks for working on this.
> 
> Do you know if the altStore app supports a button similar to playstore/appstore? Would be great to display on the landing page on github.

I just sent a pull request adding this!

**Pull request merged**